livekit-plugins/livekit-plugins-browser/livekit/plugins/browser/__init__.py

```diff
@@ -27,3 +27,12 @@ def __init__(self):
 
 
 Plugin.register_plugin(BrowserPlugin())
+
+# Cleanup docs of unexported modules
+module = dir()
+NOT_IN_ALL = [m for m in module if m not in __all__]
+
+__pdoc__ = {}
+
+for n in NOT_IN_ALL:
+    __pdoc__[n] = False
```

Comment:
I'm curious if this is really needed.
https://pdoc3.github.io/pdoc/doc/pdoc/#what-objects-are-documented

Reply:
It seems like pdoc if a directory has an __init__.py, pdoc is considering it a submodule. Even if it's not in __all__. Technically these modules are importable in a normal, non-package setup, so I guess it makes sense as a default behavior.